lms/djangoapps/courseware/tabs.py

```diff
@@ -54,30 +60,10 @@ def link_func(course, reverse_func):
         super().__init__(tab_dict)
 
     @classmethod
-    def is_enabled(cls, course, user=None):
+    def is_enabled(cls, course, user=None):  # pylint: disable=unused-argument
```

Comment:
todo: remove this method entirely since it just returns true

Reply:
> Question: Should this method be removed now that it just returns True?


I vote yes! :smile: That might put you over the 15k mark, too.